content/en/docs/reference/kubectl/docker-cli-to-kubectl.md

```diff
@@ -15,6 +15,7 @@ You can use the Kubernetes command line tool `kubectl` to interact with the API
 ## docker run
 
 To run an nginx Deployment and expose the Deployment, see [kubectl create deployment](/docs/reference/generated/kubectl/kubectl-commands#-em-deployment-em-).
+
```

Comment:
I think the change is unnecessary.

Reply:
but it's really a md format error.

Reply:
but it's really a md format issue. the fix is not bad.

Reply:
"UNNECESSARY"

Reply:
> "UNNECESSARY"

well, this is just a minor formatting issue, not a significant concern. There won't be any problems if you don't modify it. 😄. A document's readability isn't compromised by formatting alone. No need to escalate it or make it widely known. I just found the ANNOUNCING here https://www.xiaohongshu.com/explore/66b49060000000001e01eef3.

Reply:
"Unbelievable"

Reply:
How about to change title  with `fix: typo`?

Reply:
@darkokoa  @RunningFaster 
Please don't bring your disgusting "riddler culture" to Github.  
This is a place for opensource **communication**.  
If you have an advice or suggestion, express it.

Reply:
> but it's really a md format issue. the fix is not bad.

Since she wants to add a newline, she might as well use markdown lint to do it thoroughly.
Otherwise, adding just one newline is pointless.